astropy/units/quantity.py

```diff
@@ -607,6 +618,15 @@ def to(self, unit, equivalencies=[]):
         return self._new_view(new_val, unit)
 
     @property
+    def info(self):
```

Comment:
OK, fair enough. Eventually we may want to simplify this a bit to avoid needless boilerplate, but no need to do that now.